src/site/content/en/blog/css-marker-pseudo-element/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Custom Bullets with CSS ::marker
+subhead: It is now trivial to customize the color, size or type of number or bullet when using a <ul> or <ol>. 
+authors:
+  - adamargyle
+  - loirooriol
+description: It is now trivial to customize the color, size or type of number or bullet when using a <ul> or <ol>.
+date: 2020-07-27
+updated: 2020-07-27
+hero: hero.jpg
+alt: TODO
+--- 
+
+Thanks to Igalia, sponsored by Bloomberg, we can finally put our hacks away for styling lists. See!
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="marker-fun.png">
+  <figcaption class="w-figcaption">
+    <a href="https://glitch.com/edit/#!/marker-fun-example">View Source</a>
+  </figcaption>
+</figure>
+
+Thanks to [CSS `::marker`](https://www.w3.org/TR/css-lists-3/#marker-pseudo) we can change the content and some of the styles of bullets and numbers. At the time of writing `::marker` is [supported](https://caniuse.com/#search=%3A%3Amarker) in Firefox 68 for desktop and Android, Safari 11.1 and iOS Safari 11.3, and Chromium 86 (desktop and Android).
+
+## Pseudo-elements
```

Comment:
```suggestion
Thanks to [CSS `::marker`](https://www.w3.org/TR/css-lists-3/#marker-pseudo) we can change the content and some of the styles of bullets and numbers. 

## Browser compatibilty

At the time of writing `::marker` is supported in Firefox for desktop and Android, desktop Safari and iOS Safari, and Chromium-based desktop and Android browsers. See MDN's [Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/CSS/::marker#Browser_compatibility) table for updates.

## Pseudo-elements
```

Are you OK with me adding a section heading? I like to make this information very easy to find. I'm also avoiding mentioning specific versions of the browsers. I think the important thing is that it's supported in these browsers. I don't think people need to know the exact versions. If they do, they can go to MDN's table. I also prefer to link to MDN rather than caniuse for consistency. They both pull from the same source but MDN is more official.